pom.xml

```diff
@@ -64,12 +64,12 @@
     slightly-older patch, we can override the version here to the latest patch.
     See DTX-222 and DTX-312 for details.
     -->
-    <protobuf.version>3.25.3</protobuf.version>
+    <protobuf.version>4.31.1</protobuf.version>
     <build-tools.dir>${project.basedir}/build-tools</build-tools.dir>
     <spotbugs.maven.plugin.version>4.8.5.0</spotbugs.maven.plugin.version>
     <jacoco-maven-plugin.version>0.8.12</jacoco-maven-plugin.version>
     <quarkus.platform.version>3.20.0</quarkus.platform.version>
-    <io.confluent.common.version>7.7.0</io.confluent.common.version>
+    <io.confluent.common.version>8.0.0</io.confluent.common.version>
```

Comment:
Bump was needed to get access to the new deserializers that support reading the schema ID from the headers.